mod/groups/views/default/groups/profile/fields.php

```diff
@@ -3,32 +3,43 @@
  * Group profile fields
  */
 
-$group = $vars['entity'];
+$group = elgg_extract('entity', $vars);
+if (!$group instanceof ElggGroup) {
+	return;
+}
 
 $profile_fields = elgg_get_config('group');
+if (empty($profile_fields) || !is_array($profile_fields)) {
+	return;
+}
 
-if (is_array($profile_fields) && count($profile_fields) > 0) {
-	foreach ($profile_fields as $key => $valtype) {
-		// do not show the name
-		if ($key == 'name') {
-			continue;
-		}
-
-		$value = $group->$key;
-		if (is_null($value)) {
-			continue;
-		}
+foreach ($profile_fields as $key => $valtype) {
+	// do not show the name
+	if ($key == 'name') {
+		continue;
+	}
 
-		$options = ['value' => $group->$key];
-		if ($valtype == 'tags') {
-			$options['tag_names'] = $key;
-		}
+	$value = $group->$key;
+	if (is_null($value)) {
+		continue;
+	}
 
-		echo "<div>";
-		echo "<b>";
-		echo elgg_echo("groups:$key");
-		echo ": </b>";
-		echo elgg_view("output/$valtype", $options);
-		echo "</div>";
+	$options = ['value' => $group->$key];
+	if ($valtype == 'tags') {
+		$options['tag_names'] = $key;
 	}
+	
+	$field_title = elgg_echo("groups:{$key}");
+	$field_value = elgg_view("output/$valtype", $options);
+	
+	echo <<<___FIELD
+	<div class='clearfix group-profile-field'>
```

Comment:
Just add the view for this.

Reply:
Don't use clearfix. There are 3 types of containers for grids that control the responsiveness

Reply:
should be done as part of a fix for #10412.. this PR makes them at least inline with eachother